phys/module_radiation_driver.F

```diff
@@ -1042,6 +1049,15 @@ SUBROUTINE radiation_driver (                                          &
          jte = j_end(ij)
          CALL radconst(XTIME,DECLIN,SOLCON,JULIAN,               &
                        DEGRAD,DPD                                )
+
+! amontornes-bcodina 2015/09 solar eclipses
+!    Solar eclipse prediction based on the Bessel's method
+!    outputs: obscur, mask, elat_track, elon_track
+                       CALL solar_eclipse(ims,ime,jms,jme,its,ite,jts,jte,  &
```

Comment:
@irowebbn 
Isaac,
Should we only go into this routine if the sw_eclipse flag is set?

Reply:
@irowebbn 
Isaac,
Is there some sort of initial read of the data? What computations happen each time step? Are we in a portion of the code that is processed every time step or every radiation time step?